### PR TITLE
Translation update xcom game files fra int

### PR DESCRIPTION
**Pull request by Savancosinus**
Update, to fix issues and to include LWotC's new contents.
Update encoding on the ".fra" file of "LW_FactionBalance" folder.

### CODE REVIEW

File: LongWarOfTheChosen/Localization/XComGame.fra [hunk 1]
```diff
@@ -38,7 +38,7 @@ m_strLowEngineersList[2]="Un ingénieur est parfois disponible au marché noir e
 m_strRetributionAlertTitle="VENGEANCE DES ÉLUS"
 m_strRetributionAlertDescription="Un Élu limite les activités de la résistance dans cette région, ce qui réduit l'efficacité des actions entreprises dans ce secteur." 
 m_strRetributionAlertIncomeDecreasedTitle="REFUGE SABOTÉ"
-m_strRetributionAlertIncomeDescription="L'efficacité du refuge lors des missions de Ressources, Renseignements et Recruter est réduite de 20%."
+m_strRetributionAlertIncomeDescription="L'efficacité du refuge lors des missions de Ressources, Renseignements et Recruter est réduite de 20 %."
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.fra:41]: Is the extra space here intentional?

File: LongWarOfTheChosen/Localization/XComGame.fra [hunk 2]
```diff
@@ -118,6 +118,22 @@ LocFriendlyName="Position défensive"
...
+[ReturnFire X2AbilityTemplate]
+LocHelpText="Si vous êtes visé par un tir ennemi, vous ripostez automatiquement avec votre pistolet une fois par tour."
+LocLongDescription="Si vous êtes visé par un tir ennemi, vous ripostez automatiquement avec votre pistolet une fois par tour."
+
+[Quickdraw X2AbilityTemplate]
+LocLongDescription="Le tir avec votre pistolet comme première action ne met plus fin à votre tour de jeu."
+LocHelpText="Le tir avec votre pistolet comme première action ne met plus fin à votre tour de jeu."
+
+[LightningHands X2AbilityTemplate]
+LocLongDescription="Fire your pistol at a target. This attack does not cost an action."
```
Comment (pledbrook) [inline LongWarOfTheChosen/Localization/XComGame.fra:130]: Missing a translation here? :smile:

File: LongWarOfTheChosen/Localization/XComGame.fra (hunk 1 above)
Comment (Savancosinus) [reply LongWarOfTheChosen/Localization/XComGame.fra:41]: Yes. All the lines with a % in it have a space behind the char.

File: LongWarOfTheChosen/Localization/XComGame.fra (hunk 2 above)
Comment (Savancosinus) [reply LongWarOfTheChosen/Localization/XComGame.fra:130]: Oups!

**Comment by Savancosinus** [comment]
The LWotC's team added a lot of content, I'll translate it as soon as possible once you'll accept this request.

**Comment by Savancosinus** [comment]
OK, all files have been updated. 
I decided to keep original comments (even if they're irrelevant in the french version), because I want to have 2 identicals int/fra files (= same number of lines).
I also translated the following lines, but they've not been flagged with the comments "LWOTC needs translation"/"End translation" in the ".int" file (LongWarOfTheChosen/Localization/XComGame.int).
- m_strContinueTraining="
- m_strPsiTrainingCompleteHelp="

**Comment by Savancosinus** [comment]
Change the encoding (UTF-16LE).

**Comment by pledbrook** [comment]
Sorry for taking a long time off and not reviewing this yet. I'll start looking at it soon, if not today.

**Comment by pledbrook** [comment]
> I also translated the following lines, but they've not been flagged with the comments "LWOTC needs translation"/"End translation" in the ".int" file (LongWarOfTheChosen/Localization/XComGame.int).
> 
> * m_strContinueTraining="
> * m_strPsiTrainingCompleteHelp="

That's perfectly fine. Feel free to translate old LW2 stuff that is missing or incorrect or just badly translated.

**Comment by pledbrook** [comment]
> Change the encoding (UTF-16LE).
> EDIT: Did the same with the Alien Pack file. Otherwise, accents don't display properly.

This has actually broken my ability to merge your changes. The encoding should be UTF-8. It might not take too much effort to fix this, but please don't do it in future. The build converts the files from UTF-8 to UTF-16 when it packages the mod.

Not sure why accents weren't displaying correctly if the file was UTF-8. Should work fine.

**Comment by pledbrook** [comment]
I have hopefully committed all the changes [here](https://github.com/long-war-2/lwotc/commit/683b894ff8fdeee7afd3e0fc3aa53a8943e9da54). Thanks for the contribution!

**Pull request closed without merging**